### PR TITLE
FAI-168 Improve UI responsiveness

### PR DESCRIPTION
**Pull request by kelvah**
JIRA ticket: https://issues.redhat.com/browse/FAI-168

Improving responsiveness of Trusty UI.

### CODE REVIEW

**Comment by jstastny-cz** [comment]
I'm not able to review in timely manner, removing myself as a reviewer, please assess if you want to replace me with someone else (regardless their role in the team).

**Comment by winklerm** [comment]
Jenkins retest this

**Review by manstis — APPROVED** [review]
LGTM :+1:

**Comment by sjamboro** [comment]
Hi @kelvah ,
I am not sure if pagination works correctly:
image

Top: there is not able to change page or number of rows.
Bottom: works (I am able to change page and number of rows on page)

I am not sure if this behavior is expected or not. Based on https://www.patternfly.org/v4/components/pagination only bottom version works for minimal width without lost of functionality.

Feel free to add changes to this PR or create new issue etc. Just let me know. Other parts of this PR seems ok.

File: ui-packages/packages/trusty/src/components/Molecules/SkeletonCards/SkeletonCards.tsx [hunk 1]
```diff
@@ -17,11 +17,11 @@ const SkeletonCards = (props: SkeletonCardProps) => {
         <div className="skeleton-cards__card">
           <SkeletonStripe
             key={uuid()}
-            customStyle={{ width: 250, height: 25, marginBottom: '20px' }}
+            customStyle={{ width: '90%', height: 25, marginBottom: '20px' }}
```
Comment (sjamboro) [inline ui-packages/packages/trusty/src/components/Molecules/SkeletonCards/SkeletonCards.tsx:20]: @kelvah can you give me a tip where I can find this part in UI?

Comment (kelvah) [reply ui-packages/packages/trusty/src/components/Molecules/SkeletonCards/SkeletonCards.tsx:20]: @sjamboro sure, they are the placeholder cards that can be briefly seen in the Outcomes page of an execution while the data is fetched.

**Comment by kelvah** [comment]
> Hi @kelvah ,
> I am not sure if pagination works correctly:

Hi @sjamboro, yes, there was a small issue on mobile resolutions that I decided to ignore :upside_down_face: .

PF's bottom navigation should be always visible on mobile because it's the only one that can be used to change page. In order to avoid having to scroll all the way down, It should be absolutely positioned to the bottom of the screen (actually it's `position: sticky`).

This positioning wasn't working though because the page container didn't have an explicit `height`. I've fixed it now.

**Comment by kelvah** [comment]
Hi @sjamboro, do you have any other comments on the PR?

**Comment by kelvah** [comment]
The 4 sonar bugs are basically `sonar-css` not recognizing [sass `@use`](https://sass-lang.com/documentation/at-rules/use):

> Unexpected unknown at-rule "@use"

It seems [it was fixed](https://github.com/SonarSource/sonar-css/pull/326) a couple of months ago, but we still have it.

**Review by sjamboro — APPROVED** [review]

**Comment by sjamboro** [comment]
@kelvah I am sorry, this PR disappears from my "review request" list. I hope that everything is ok now

**Comment by kelvah** [comment]
> The 4 sonar bugs are basically `sonar-css` not recognizing [sass `@use`](https://sass-lang.com/documentation/at-rules/use):
> 
> > Unexpected unknown at-rule "@use"
> 
> It seems [it was fixed](https://github.com/SonarSource/sonar-css/pull/326) a couple of months ago, but we still have it.

Hi @manstis, @danielezonca, how do you suggest to proceed?

**Comment by jstastny-cz** [comment]
@kelvah is there any value in scss files being included in sonar analysis? If not we can tweak inclusions/exclusions for the import.

**Comment by kelvah** [comment]
@jstastny-cz I don't have a strong opinion. 

having static analysis for sass/css is a good thing but `sonar-css` seems a bit weak on the sass side.
and the bug I've found [took a whole year](https://github.com/SonarSource/sonar-css/issues/252) to get fixed...  

tough call I guess. if it's feasible we can perhaps disable just the [at-rule check](https://sonarcloud.io/organizations/kiegroup/rules?activation=true&open=css%3AS4662&qprofile=AWQmnmafHlEhtnAoG8H2) and give it another chance?

**Comment by manstis** [comment]
> > The 4 sonar bugs are basically `sonar-css` not recognizing [sass `@use`](https://sass-lang.com/documentation/at-rules/use):
> > > Unexpected unknown at-rule "@use"
> > 
> > 
> > It seems [it was fixed](https://github.com/SonarSource/sonar-css/pull/326) a couple of months ago, but we still have it.
> 
> Hi @manstis, @danielezonca, how do you suggest to proceed?

I suggest this is merged and another PR submitted by someone relaxing the `@use` check.

**Comment by kelvah** [comment]
thank you @manstis.

@jstastny-cz, @danielezonca any objections? Otherwise I would proceed and merge this one.

**Comment by jstastny-cz** [comment]
@kelvah sounds good. Please go ahead with merge and track the task to exclude the files or relax the rule in future.

**Comment by kelvah** [comment]
Issue tracked here https://issues.redhat.com/browse/KOGITO-5020

**Pull request merged**